extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointMMSClient.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class ResourceLoaderAdEngineSourcePointMMSClient extends ResourceLoaderAdEngineSourcePointBase {
+	// increase this any time the local files change
+	const CACHE_BUSTER = 0;
+	const SCRIPT_URL = 'https://api.sourcepoint.com/script/mms_client?v=1';
+	const FALLBACK_SCRIPT_URL = __DIR__ . '/../SourcePoint/mms_client.js';
+
+	/**
+	 * Fallback data when request to external script fails
+	 * @return array ["script" => '', "modTitme" => '', "ttl" => '']
+	 */
+	protected function getFallbackDataWhenRequestFails() {
+		$scripts = [];
+		$scripts[] = (new ResourceLoaderScript())->setTypeLocal()->setValue(self::FALLBACK_SCRIPT_URL);
+		$data = [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$data". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)